python/tops2vrt.py

```diff
@@ -248,13 +215,13 @@ def getLinePixelBbox(geobbox, latFile, lonFile):
     print('write vrt file for geometry dataset')
     layers = ['lat', 'lon', 'hgt']
     for ind, val in enumerate(layers):
-        with open( os.path.join(inps.geomdir, val+'.vrt'), 'w') as fid:
-            fid.write( vrttmpl.format( xsize = xsize, ysize = ysize,
-                                       xmin = xmin, ymin = ymin,
-                                       width = width,
-                                       height = height,
-                                       PATH = os.path.abspath( os.path.join(inps.indir, 'geom_reference', val+'.rdr.full.vrt')),
-                                       linewidth = width * 8))
-
-
-
+        with open(os.path.join(inps.geomdir, val + '.vrt'), 'w') as fid:
+            fid.write(vrttmpl.format(xsize=xsize, ysize=ysize,
+                                     xmin=xmin, ymin=ymin,
+                                     width=width,
+                                     height=height,
+                                     PATH=os.path.abspath(
+                                         os.path.join(inps.indir,
+                                                      'geom_reference',
+                                                      val + '.rdr.full.vrt')),
+                                     linewidth=width * 8))
```

Comment:
To increase readability shall we define a variable (e.g, PixelOffset=8 # bytes) and then change this line to 
```
linewidth=width * PixelOffset))
```